plugins/uberdev/lib/goal-watch.sh

```
@@ -0,0 +1,1022 @@
+  # NO gh call at all — exactly as the pre-#301 per-issue finder did — and the
+  # #290.3 consecutive-failure counter is free to accumulate the 2c/2d
+  # `gh pr view` failures that are the only gh traffic left. Reset it here every
+  # pass and a sustained `gh pr view` outage is pinned at 1 forever: the
+  # gh_api_failed breaker can never fire and /goal rides the 4h stuck_loop
+  # instead of halting loudly in ~5 min. Lazy also keeps the steady-state API
+  # cost at zero rather than one 200-PR page per pass (up to 6/min under the
+  # made_transition fast path).
+  pr_snapshot=""
+  _pr_snapshot_taken=0
+
+  for issue in "${active_issues[@]}"; do
```

Shellcheck (0.11.0)

[warning] active_issues is referenced but not assigned.

(SC2154)